x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -133,6 +134,18 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
       }),
     ],
   },
+  {
+    id: SecurityPageName.manageLanding,
```

Comment:
i thought we were going to delete this and name it `SecurityPageName.administration`?

Reply:
yep, replaced